LHC/Injection_Energy_with_Misalignments_and_Correction/README.md

```diff
@@ -3,3 +3,5 @@ Misalignments are applied and orbit is corrected.
 Nonlinear errors are applied and corrections applied as were used in RunII.
 A1/A2/B1/B2 are excluded as these are assumed to have been well corrected and residual betabeating and
 coupling stems for misalignments and feed down.
+
+Set `NRJ`?
```

Comment:
NRJ should be kept a t 450 as technically this script is only valid for injection energy, at top energy the magnetic errors of the triplet play a bigger role and nonlinear correctors should be set too, which is not done in this script (but there is an open issue to add a script for that in MESS)

Reply:
Good catch.